Sources/MockoloFramework/Models/MethodModel.swift

```diff
@@ -16,85 +16,145 @@
 
 import Foundation
 import SourceKittenFramework
+import SwiftSyntax
 
 final class MethodModel: Model {
+    
     var filePath: String = ""
+    var data: Data? = nil
     var name: String
     var type: Type
     var offset: Int64
     let length: Int64
     let accessControlLevelDescription: String
     let attributes: [String]
-    let staticKind: String
-    var genericTypeParams: [ParamModel] = []
-    var params: [ParamModel] = []
-    let handler: ClosureModel?
+    let genericTypeParams: [ParamModel]?
```

Comment:
Seem asymmetric for this to be optional and the next one not. Can't the callers just pass in `[]` since that's what will happen later anyway?

Reply:
I changed it to be optional because swiftsyntax treats also this as an optional property. Is there a more preferred way: handle an optional within this class vs. at the caller?

Reply:
made it nonoptional